CHANGELOG/v1.4.md

```diff
@@ -12,15 +12,23 @@ Non-backwards compatible changes
 
 ### Changes to binary trees
 
-* Binary trees now have two parameters, the first being the type of node labels, and the second being the type of leaf labels.
+* Binary trees now have two parameters, the first being the type of node
+  labels, and the second being the type of leaf labels.
 * **Changes Required**
-Instead of using `Tree A` from `Data.Tree.Binary`, you now must use `Tree A ⊤` to get the same functionality.
+Instead of using `Tree A` from `Data.Tree.Binary`, you now must use
+`Tree A ⊤` to get the same functionality.
 
-`map` now requires a function on both nodes and leaves. There are also `mapₙ` and `mapₗ` which only map on nodes and leaves respectively. There are similar changes for `Data.Tree.Relation.Unary.All`.
+`map` now requires a function on both nodes and leaves. There are also
+`mapₙ` and `mapₗ` which only map on nodes and leaves respectively. There
+are similar changes for `Data.Tree.Relation.Unary.All`.
 
 Deprecated modules
 ------------------
 
+* In `Algebra.Morphism.Definitions` and `Relation.Binary.Morphism.Definitions`
+  the type `Morphism A B` were recovered by publicly importing its
+  definition from `Function.Core`. See discussion in issue #1206.
```

Comment:
This would more appropriately live under `Deprecated names` rather than `Deprecated modules`